packages/xs-vat-worker/start-xs.js

```diff
@@ -0,0 +1,45 @@
+/* global trace */
+
+// eslint-disable-next-line import/no-unresolved
+import Timer from '@moddable/timer';
+// eslint-disable-next-line import/no-unresolved
+import Resource from '@moddable/Resource';
+import { makeHandledPromise } from '@agoric/eventual-send';
+
+import { harden } from './src/harden';
+import { makeConsole } from './src/console';
+import { loadMain } from './src/endo-load';
+
+// eslint-disable-next-line import/named
+import { Reader, Writer } from './src-native/fdchan';
+
+const INFD = 3;
```

Comment:
we should probably make these configurable (a single kernel process will be running multiple xs-workers, and they'll each need a separate set of FDs I think), but it can wait

Reply:
funny... when last we spoke I'm pretty I asked you if each vat would need its own pair of FDs and you said no... **oh**... multiple xs-workers. gotcha.

Reply:
right, multiple vats per process, multiple processes, one pipe-pair per process (not one per vat)

Reply:
postponed as https://github.com/Agoric/agoric-sdk/issues/1616